Misc/reputation-ip.json

```diff
@@ -7,21 +7,22 @@
     "commitMessage": "",
     "shouldPublish": false,
     "shouldCommit": false,
-    "regex": "\\b(?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\b",
+    "regex": "\\b(?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])(?:\\[\\.\\]|\\.)){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\b",
     "details": "IP",
     "prevDetails": "IP",
     "reputationScriptName": "",
     "reputationCommand": "ip",
     "enhancementScriptNames": [
-        "IPReputation"
+        "IPReputation",
+        "SplunkPySearch"
```

Comment:
Please check why SplunkPySearch was in escaped

Reply:
Previously there was `SplunkSearch` under IP and IPescaped, but it was removed from IP and replaced in IPescaped with `SplunkPySearch` in https://github.com/demisto/content/pull/2931

Reply:
Please add the SplunkPySearch command to the new IP